src-java/kilda-pce/src/test/java/org/openkilda/pce/AvailableNetworkFactoryTest.java

```diff
@@ -272,4 +433,14 @@ private static void assertAvailableNetworkIsCorrect(IslImmutableView isl, Availa
         assertNotNull(dst);
         assertEquals(1, dst.getIncomingLinks().size());
     }
+
+    private static void assertAvailableNetworkIsNotCorrect(IslImmutableView isl, AvailableNetwork availableNetwork) {
+        Node src = availableNetwork.getSwitch(isl.getSrcSwitchId());
+        assertNotNull(src);
+        assertEquals(0, src.getOutgoingLinks().size());
+
+        Node dst = availableNetwork.getSwitch(isl.getDestSwitchId());
+        assertNotNull(dst);
+        assertEquals(0, dst.getIncomingLinks().size());
```

Comment:
It's not quite correct. this assert is valid only for your test case. To make it more correct you need to go throw all 
```
src.getOutgoingLinks()
src.getIncomingLinks()
dst.getOutgoingLinks()
dst.getIncomingLinks()
```

And check that all of them has no ISL `isl`

Reply:
Done, checked it inside the test case